data/scripts/talkactions/position.lua

```diff
@@ -0,0 +1,15 @@
+local talk = TalkAction("/pos")
+
+function talk.onSay(player, words, param)
+	if player:getGroup():getAccess() and param ~= "" then
+		local split = param:split(",")
+		player:teleportTo(Position(split[1], split[2], split[3]))
+	else
+		local position = player:getPosition()
+		player:sendTextMessage(MESSAGE_STATUS_CONSOLE_BLUE, "Your current position is: " .. position.x .. ", " .. position.y .. ", " .. position.z .. ".")
+	end
+	return false
+end
+
+talk:separator(" ")
+talk:register()
```

Comment:
newline

Reply:
fixed in [9c085d8](https://github.com/otland/forgottenserver/pull/2558/commits/9c085d844c9bc73bed7f6056131be74ed4a6ce03)